101-azure-bastion/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vnet-name": {
+      "value": "vnet01"
```

Comment:
remove the non-GEN values from here and just use defaultValues in the template

Reply:
done. I left "ussouthcentral" for location in the parameters file to ensure CI testing, since the preview is only available in a subset of regions currently.